app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryPresenter.java

```diff
@@ -33,6 +34,7 @@ void loadBooks() {
         .subscribe(library -> {
           getMvpView().showBooks(library.getBooks());
         }, error -> {
+          Log.w("kiwixLibrary", error.getLocalizedMessage());
```

Comment:
error.getLocalizedMessage() seems to be an empty String/ null in some cases and this causes and app crash as seen on testdroid.